CHANGELOG.md

```diff
@@ -10,6 +10,111 @@ Note: Changes are now automatically tracked in [GitHub](https://github.com/sqlfl
 -->
 <!--Start Of Releases (DO NOT DELETE THIS LINE)-->
 
+## [1.3.0] - 2022-08-21
+
+## Highlights
+
+This release brings several potentially breaking changes to the underlying parse tree. For
```

Comment:
Suggest adding:
```
* New command-line option `--show-lint-violations` to show details on unfixable errors when running `sqlfluff fix`
* Improve consistency of process exit codes
* Jinja templater: When `--ignore=templating` is enabled, undefined Jinja variables now take on "reasonable" default values rather than blank string (`""`). This can streamline initial rollout of SQLFluff by reducing or eliminating the need to configure templater variables.